Doc/library/functions.rst

```diff
@@ -168,6 +168,11 @@ are always available.  They are listed here in alphabetical order.
    If :func:`sys.breakpointhook` is not accessible, this function will
    raise :exc:`RuntimeError`.
 
+   By default (if :func:`sys.breakpointhook` has not been replaced), the
+   behavior of :func:`breakpoint` can be changed with the
+   :envvar:`PYTHONBREAKPOINT` environment variable.  See
+   :func:`sys.breakpointhook` for details.
```

Comment:
An attempt to remove the bracked aside and move the less common case outside of the main clause:
```suggestion
   By default, the behavior of :func:`breakpoint` can be changed with
   the :envvar:`PYTHONBREAKPOINT` environment variable.
   See :func:`sys.breakpointhook` for usage details.
   
   Note that this  is not guaranteed if :func:`sys.breakpointhook`
   has been replaced.
```